src/components/app/app.js

```diff
@@ -3,7 +3,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import { ThemeProvider } from '@material-ui/styles';
 import { CircleArrow as ScrollUpButton } from 'react-scroll-up-button';
 import { CssBaseline } from '@material-ui/core';
-
+import { getBurgerMenu } from '../../redux/burgerMenu/burgerMenu.actions';
```

Comment:
Rename this to `getBurgerMenuLinks` or something like this

Reply:
Renamed